source/extensions/filters/http/ip_tagging/ip_tagging_filter.cc

```diff
@@ -112,6 +108,46 @@ void IpTaggingFilter::setDecoderFilterCallbacks(Http::StreamDecoderFilterCallbac
   callbacks_ = &callbacks;
 }
 
+void IpTaggingFilter::applyTags(Http::RequestHeaderMap& headers,
+                                const std::vector<std::string>& tags) {
+  using HeaderAction = IpTaggingFilterConfig::HeaderAction;
+
+  OptRef<const Http::LowerCaseString> header_name = config_->ip_tag_header();
+
+  if (tags.empty()) {
+    bool mustSanitize =
```

Comment:
this isn't really mustSanitize given we'll only do it if there's a header name configured.  Also snake case for Envoy variables
maybe_sanitize?

Reply:
(I wish there was a linter for this.) <3